adapters/adagio/adagio_test.go

```diff
@@ -54,8 +54,8 @@ func TestMakeRequests_NoGzip(t *testing.T) {
 	assert.Equal(t, 1, len(requestData))
 
 	body := &openrtb2.BidRequest{}
-	_ = json.Unmarshal(requestData[0].Body, body)
-
+	error := json.Unmarshal(requestData[0].Body, body)
```

Comment:
Can you please rename this to `err` rather? `error` is a reserved keyword in Go and can therefore be confusing.